### DISCUSSION TITLE
Huge elastalert_status index

### DISCUSSION THREAD

**Discussion by 3isenHeiM** [Security-Onion-Solutions/securityonion#9827] [Unsupported Versions]
On my manager node, the `elastalert_status` indices are huge.

![image](https://user-images.githubusercontent.com/26417172/220319165-f25157a9-e622-4cfc-97a2-1105acb1de10.png)

How can I reduce them?

**Comment by eb-op** [comment]
Hello 3isenHeiM,

Could I get a little more information on your issue? How many elastalert rules do you have enabled, and how much data are you pulling from them? That seems to be what is filling up the `elastalert_status` index.
